lib/cli/stats.js

```diff
@@ -159,48 +159,67 @@ async function setMainPath(templateType, templateName) {
   return filePath;
 }
 
+function percentageRoundTwo(num, den) {
+  return den > 0 ? Number(((num / den) * 100).toFixed(2)) : 0;
+}
+
 async function getTemplatesSummary() {
   const summary = {
     reconciliations: {
       total: 0,
       externallyManaged: 0,
+      externallyManagedPerc: 0,
       yamlFiles: 0,
+      yamlFilesPerc: 0,
       unitTests: 0,
     },
     sharedParts: {
       total: 0,
       externallyManaged: 0,
+      externallyManagedPerc: 0,
     },
     exportFiles: {
       total: 0,
       externallyManaged: 0,
+      externallyManagedPerc: 0,
     },
     accountTemplates: {
       total: 0,
       externallyManaged: 0,
+      externallyManagedPerc: 0,
       yamlFiles: 0,
+      yamlFilesPerc: 0,
       unitTests: 0,
     },
     all: {
       total: 0,
       externallyManaged: 0,
+      externallyManagedPerc: 0,
       yamlFiles: 0,
+      yamlFilesPerc: 0,
       unitTests: 0,
     },
   };
 
   // Reconciliations
-  const reconciliationsNonEmpty = await listNonEmptyTemplates(
-    "reconciliationText"
-  );
+  const reconciliationsNonEmpty =
```

Comment:
[SUGGESTION] Restore assignment to 1 line for consistency with other assign statements?

Reply:
Multi-line in this assignment was automatically implemented by the formatter (prettier). I  tend to respect them if not we will need to manually adjust them constantly.

FYI We could consider modifying the configuration of the formatter as we want, so far we only have two rules defined https://github.com/silverfin/silverfin-cli/blob/7d9ce941a15aa361753c59e6afba713a50a79996/package.json#L39

Reply:
Happy to respect the formatter